apps/dot-voting/app/store/votes.js

```diff
@@ -226,8 +229,69 @@ const marshallVote = ({
     totalParticipation: totalParticipation,
     metadata,
     executionScript,
+    executionTargetData,
     executed,
     participationPct:
       totalVoters === 0 ? 0 : (totalParticipation / totalVoters) * 100,
   }
 }
+
+const getVoteExecutionTargets = (vote) => {
+  return vote.executionTargets || []
+}
+
+const decorateVote = async (vote) => {
+  const executionTargets = getVoteExecutionTargets(vote)
+  const currentApp = await app.currentApp().toPromise()
+  if (!executionTargets.length) {
+    // If there's no execution target, consider it targetting this Dot Voting app
+    return {
+      ...vote,
+      executionTargetData: {
+        ...currentApp,
+        identifier: undefined,
+      }
+    }
+  } else if (executionTargets.length > 1) {
+    // If there's multiple targets, make a "multiple" version
+    return {
+      ...vote,
+      executionTargetData: {
+        address: '0x0000000000000000000000000000000000000000',
+        name: 'Multiple',
+        iconSrc: currentApp.icon(24),
+        identifier: undefined,
+      }
+    }
```

Comment:
We might want to consider just defaulting to the last app in the target chain here, but this will require more discussion around forwarding design, and if that satisfies all use cases. Otherwise, we should consider a popout that shows the remaining execution chain or something more useful than just saying `Multiple`.